### DISCUSSION TITLE
Add audio output format

### DISCUSSION THREAD

**Discussion by alexpado** [alexpado/genshin-music-unpacker#9] [Ideas]
I don't know if people would be interested about it, but the idea would be to add ffmpeg to convert WAV file to another format like MP3 or OGG.

**Comment by pha1n0q** [comment]
Is it possible to have direct conversion of the WEM files to FLAC (I'd love for FLAC to be implemented due to it's compression!), MP3, and/or OGG (I know that ffmpeg can do this, but not sure about VGMStream)? This would reduce the amount of computing power/time to convert from WEM > WAV > other formats.

**Reply by alexpado** [reply]
I can check later if vgmstream can convert into something else than WAV. I know it supports FLAC other format, but I don't know if it includes FLAC for export specifically.

If it's not possible, I can still include ffmpeg to convert WAV into other audio format
